libraries/radspreadprocessing/cross-platform-support/text-measure.md

```diff
@@ -0,0 +1,66 @@
+
+This topic describes the default and the extended implementations used to measure text while exporting to PDF.
+
+## Using a Text Measurer
+
+The **SpreadExtensibilityManager** static class exposes the **TextMeasurer** property. This property defines the specific implementation that is used to measure the text content when needed. You can use the following values for this property:
+
+- [SimpleTextMeasurer](#simpletextmeasurer): This is the default value.
+- [SpreadFixedTextMeasurer](#spreadfixedtextmeasurer): An additional implementation that provides more accurate results than the SimpleTextMeasurer.
+- [Custom implementation](#custom-text-measurer): Implementing the abstract SpreadTextMeasurerBase class enables you to provide your own implementation for text measuring.
+
+## SimpleTextMeasurer
+
+This is the measurer used by default when exporting to PDF.
+
+The width of the columns in Excel is stored as a character count calculated based on the maximum width of the digits from 0 to 9 with the Normal Style of the document applied to them. SimpleTextMeasurer uses the measured width of the digits for a predefined set of font families to calculate the column width based on the length of the text multiplied by the calculated character size. 
+
+* **TextMeasurer**: Gets or sets a *SpreadTextMeasurerBase* instance used to provide text measuring. The TextMeasurer has a *SimpleTextMeasurer* as a default value.
+
+> The SimpleTextMeasurer provides basic functionality for text measuring and the results might not be satisfying in each case. For better results, use [SpreadFixedTextMeasurer](#spreadfixedtextmeasurer).
+
+## SpreadFixedTextMeasurer
+
+This implementation uses PdfProcessing to obtain the size of the text and provides great precision. You need to explicitly set it to the **TextMeasurer** property of **SpreadExtensibilityManager**.
+
+>To use this class, you must add a reference to **Telerik.Documents.Fixed.dll**.
+
+#### [C#] Example 1: Set custom implementation inheriting the SpreadTextMeasurerBase abstract class
+
+{{region cs-radspreadprocessing-cross-platform_2}}
+
+    SpreadTextMeasurerBase fixedTextMeasurer = new SpreadFixedTextMeasurer();
+    SpreadExtensibilityManager.TextMeasurer = fixedTextMeasurer;
+{{endregion}}
+
+## Custom Text Measurer
+
+You can assign any **SpreadTextMeasurerBase** implementation to the **SpreadExtensibilityManager.TextMeasurer** property. All you should do is to inherit the abstract **SpreadTextMeasurerBase**, implement the required members and set the new implementation to the TextMeasurer property.
+
+#### **[C#] Example 2: Set custom implementation inheriting the SpreadTextMeasurerBase abstract class**
+{{region cs-radspreadprocessing-cross-platform_2}}
+
+    SpreadTextMeasurerBase customTextMeasurer = new TextInfo();
+    SpreadExtensibilityManager.TextMeasurer = customTextMeasurer;
+{{endregion}}
+
```

Comment:
Example 3: Here we could add a sample implementation of this TextInfo class. For example: https://docs.telerik.com/devtools/document-processing/knowledge-base/create-custom-text-measurer-net-standard